### PR TITLE
Export typedefs for protos

### PR DESCRIPTION
**Pull request by maximpertsov**
We need to copy `*.d.ts` files that we generate over to dist, since `tsc` and `vite-plugin-dts` ignore any `.d.ts` files in source apparently.

### CODE REVIEW

**Comment by maximpertsov** [comment]
Confirming that proto typedefs are available with this package:

image

**Review by micheal-parks — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   "scripts": {
     "build": "vite build",
```
Comment (mcous) [inline package.json:25]: Should this be part of the existing `build` script?

```suggestion
    "build": "vite build && npm run copy-dts",
```

File: Makefile [hunk 2]
```diff
@@ -15,6 +15,7 @@ teardown:
 
 .PHONY: build
 build: build-buf build-js
+	npm run copy-dts
```
Comment (mcous) [inline Makefile:18]: I would add this to the `build-js` target (or add it to the npm `build` script, as in my other comment)

**Review by mcous — APPROVED** [review]
LGTM, though I might move some stuff around slightly

File: Makefile (hunk 2 above)
Comment (maximpertsov) [inline Makefile:18]:
```suggestion
```

**Pull request merged**